external-crates/move/crates/move-analyzer/editors/code/scripts/create.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/zsh
+# Copyright (c) Mysten Labs, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# This script is meant to be executed on MacOS (hence zsh use - to get associative arrays otherwise
+# unavailable in the bundled bash version).
+
+set -e
+
+usage() {
+    >&2 echo "Usage: $0 -pkg|-pub [-h]"
+    >&2 echo ""
+    >&2 echo "Options:"
+    >&2 echo " -pub          Publish extensions for all targets"
+    >&2 echo " -pkg          Package extensions for all targets"
+    >&2 echo " -h            Print this message"
+}
+
+clean_tmp_dir() {
+  test -d "$TMP_DIR" && rm -fr "$TMP_DIR"
+}
+
+if [[ "$@" == "" ]]; then
+    usage
+    exit 1
+fi
+
+for cmd in "$@"
+do
+    if [[ "$cmd" == "-h" ]]; then
+        usage
+        exit 0
+    elif [[ "$cmd" == "-pkg" ]]; then
+        OP="package"
+        OPTS="-omove-VSCODE_OS.vsix"
+    elif [[ "$cmd" == "-pub" ]]; then
+        OP="publish"
+        OPTS=""
+    else
+        usage
+        exit 1
+    fi
+done
+
+# these will have to change if we want a different network/version
+NETWORK="testnet"
+VERSION="1.13.0"
```

Comment:
Could we expose these as env vars or arguments (likely we'll want to target `mainnet` and latest release?). But I'm assuming that will be handled in follow up PR, so just commenting on what I see :)

Reply:
Actually, I don't want to expose them. This script is for internal use only and we will not be updating VERSION and NETWORK at each release. In some sense, this serves as a documentation on what binaries version we are bundling with the current extension version.

That being said, we indeed do want to change it to the newest version of mainnet binaries before publishing :-)